DOC-54404-GitOps Agent Proxy Support (#1390)
harness/developer-hub: docs/continuous-delivery/gitops/install-a-harness-git-ops-agent.md

Click **New GitOps Agent**. The Agent wizard appears.

> ## Harness GitOps Agent with no existing Argo CD project

Reviewer: ```suggestion
## Harness GitOps Agent without an existing Argo CD project
```

For steps on setting up the mapping and import, go to [Map Argo projects to Harness GitOps Projects](multiple-argo-to-single-harness.md).

## Proxy support

> The GitOps Agent can work on environments where traffic is routed through a proxy. Perform the following steps to configure proxy support for GitOps Agent.

Reviewer: ```suggestion
The Harness GitOps Agent can work on environments where traffic is routed through a proxy. Perform the following steps to configure proxy support for the agent.
```

> 1. Make sure that the Agent is running in HTTP mode.  

Reviewer: ```suggestion
1. Make sure that the agent is running in HTTP mode.  
```

>    To verify, check if the property/config `GITOPS_SERVICE_PROTOCOL` value is set to `HTTP1` in the `configmap({agentname}-agent)` present in the YAML you get after creating the Agent.  

Reviewer: ```suggestion
   To verify, check if the property/config `GITOPS_SERVICE_PROTOCOL` value is set to `HTTP1` in the `configmap({agentname}-agent)` present in the YAML after you create the agent.  
```

   `GITOPS_SERVICE_PROTOCOL: HTTP1`

> 2. Add a property/config `HTTPS_PROXY` and add proxy details such as URL, port, and auth details as its value in the configmap mentioned in Step 1. For example, `HTTPS_PROXY: "https://squid.proxy-test:3128"`.

Reviewer: ```suggestion
2. Add a property/config `HTTPS_PROXY`, and add proxy details, such as URL, port, and auth details as its value in the configmap mentioned in Step 1. For example, `HTTPS_PROXY: "https://squid.proxy-test:3128"`.
```

> 3. Add an environment variable `NO_PROXY` in the GitOps Agent deployment with the following value.  

Reviewer: ```suggestion
3. Add an environment variable `NO_PROXY` in the Harness GitOps Agent deployment with the following value.  
```
